kevm-pyk/src/kevm_pyk/__main__.py

```diff
@@ -600,16 +599,13 @@ def exec_run(
     schedule: str,
     mode: str,
     chainid: int,
-    target: str | None = None,
+    target: KompileTarget,
```

Comment:
`KompileTarget` is now just used for `kevm run` and `kevm kast`, because it allows specifying which KEVM precompiled definition to use for those commands. It's no longer used for kompiling specifications (`kevm kompile ...` which became `kevm kompile-spec ...`).

Reply:
Back to `str` which is among hardcoded values.